src/commands/lint/mod.rs

```diff
@@ -73,13 +86,15 @@ pub fn lint(args: &mut LintArgs) -> anyhow::Result<()> {
 
     let mut cmd = crate::external_cli::CommandExt::new("bevy_lint");
 
-    #[cfg(feature = "rustup")]
-    cmd.require_package(parse_required_package()?);
-
     cmd.args(cargo_args)
         .env("RUSTFLAGS", args.cargo_args.common_args.rustflags.clone());
 
-    let status = cmd.ensure_status(args.auto_install())?;
+    let status = crate::external_cli::CommandExt::new("bevy_lint")
+        // We do not want to automatically install a `bevy_lint` version.
+        // The reason is that it's currently not possible to easily install
+        // the latest release without performing network requests.
```

Comment:
The phrasing "without performing network requests" is confusing to me. Don't we use network requests to auto-install other software, like `wasm-opt`?